webui/react/src/pages/DesignKit.tsx

```diff
@@ -2365,17 +2368,17 @@ const DesignKit: React.FC = () => {
           <ul>
             {componentOrder.map((componentId) => (
               <li key={componentId}>
-                <Link reloadDocument to={`#${componentId}`}>
-                  {ComponentTitles[componentId]}
-                </Link>
+                <a href={`#${componentId}`}>{ComponentTitles[componentId]}</a>
```

Comment:
**question:** was this change to make it easier to match the links?
```
const links = await page.locator('nav ul a').all();
```

Reply:
**question:** when `isExclusiveMode` is true, do we need to preserve it and keep passing `exclusive=true` via other links like these `<a>` links?

Reply:
We made this change so the `isExclusiveMode` query param is preserved when clicking on links. The `Link` element doesn't keep the search part of the url when navigating to a hash.

As other links aren't used in generating the screenshots and the intent of `exclusive` is mainly to generate screenshots, using the same treatment isn't necessary, but I can update it for consistency's sake.